src/test/regress/expected/view_propagation.out

```diff
@@ -359,13 +359,93 @@ ERROR:  cannot change name of view column "id" to "a"
+ALTER TABLE alter_view_1 ALTER COLUMN val1 DROP DEFAULT;
+ERROR:  Citus doesn't support setting or resetting default values for a column of view
+-- Set/reset options view alter view/alter table commands
+ALTER VIEW alter_view_1 SET (check_option=cascaded);
+ALTER VIEW alter_view_1 SET (security_barrier);
+ALTER VIEW alter_view_1 SET (check_option=cascaded, security_barrier);
+ALTER VIEW alter_view_1 SET (check_option=cascaded, security_barrier = true);
+ALTER TABLE alter_view_1 SET (check_option=cascaded);
+ALTER TABLE alter_view_1 SET (security_barrier);
+ALTER TABLE alter_view_1 SET (check_option=cascaded, security_barrier);
+ALTER TABLE alter_view_1 SET (check_option=cascaded, security_barrier = true);
+\d+ alter_view_1
+                    View "view_prop_schema.alter_view_1"
+ Column |  Type   | Collation | Nullable | Default | Storage  | Description
+---------------------------------------------------------------------
+ id     | integer |           |          |         | plain    |
+ val1   | text    |           |          |         | extended |
+View definition:
+ SELECT alter_view_table.id,
+    alter_view_table.val1
+   FROM alter_view_table;
+Options: check_option=cascaded, security_barrier=true
+
+\c - - - :worker_1_port
+SET search_path to view_prop_schema;
+\d+ alter_view_1
+                    View "view_prop_schema.alter_view_1"
+ Column |  Type   | Collation | Nullable | Default | Storage  | Description
+---------------------------------------------------------------------
+ id     | integer |           |          |         | plain    |
+ val1   | text    |           |          |         | extended |
+View definition:
+ SELECT alter_view_table.id,
+    alter_view_table.val1
+   FROM alter_view_table;
+Options: check_option=cascaded, security_barrier=true
+
+\c - - - :master_port
+SET search_path to view_prop_schema;
+ALTER TABLE alter_view_1 RESET (check_option, security_barrier);
+ALTER VIEW alter_view_1 RESET (check_option, security_barrier);
+-- Change the schema of the view
+ALTER TABLE alter_view_1 SET SCHEMA view_prop_schema_inner;
+ALTER VIEW view_prop_schema_inner.alter_view_1 SET SCHEMA view_prop_schema;
+-- Rename view and view's column name
+ALTER VIEW alter_view_1 RENAME COLUMN val1 TO val2;
+ALTER VIEW alter_view_1 RENAME val2 TO val1;
+ALTER VIEW alter_view_1 RENAME TO alter_view_2;
+ALTER TABLE alter_view_2 RENAME COLUMN val1 TO val2;
+ALTER TABLE alter_view_2 RENAME val2 TO val1;
+ALTER TABLE alter_view_2 RENAME TO alter_view_1;
+-- Alter owner vith alter view/alter table
+SET client_min_messages TO ERROR;
+CREATE USER alter_view_user;
+SELECT run_command_on_workers($$CREATE USER alter_view_user;$$);
+      run_command_on_workers
+---------------------------------------------------------------------
+ (localhost,57637,t,"CREATE ROLE")
+ (localhost,57638,t,"CREATE ROLE")
+(2 rows)
+
+ALTER VIEW alter_view_1 OWNER TO alter_view_user;
```

Comment:
(a) can you query the view with the non-superuser?  (b) can you create a new view with non=superuser [already added that comment to the other PR, choose which one you want to add the test]?

Those are as important as the rest of the implementation :)